setup.py

```diff
@@ -265,7 +265,8 @@ def setup_package():
                     package_data={'': ['*.pxd']},
                     **extra_setuptools_args)
 
-    commands = [arg for arg in sys.argv[1:] if not arg.startswith('-')]
+    commands = [arg for arg in sys.argv[1:] if not arg.startswith('-') and
+                not arg.startswith('/')]
```

Comment:
Does the `/` come from a file-path or something like that? Will this also work on Windows where the file separator is `\`?

Eitherway it would be good to add a small comment

Reply:
On Windows filenames would even typically start with `r"C:\\"`, `r"D:\\"` to `r"Z:\\"` as often used for shared folders on virtual machines... (I guess `r"A:\\"` and `r"B:\\"` also would be possible). And maybe `r"\\"` as well but I am not 100% sure.

Reply:
Right sorry, I'm not very good at Windows fixes... This will definitely not work with Windows. I will try harder.